public/stylesheets/manageFeedback.css

```diff
@@ -82,6 +82,14 @@ a.btn.btn-sm.btn-primary.btn-create-feed:hover {
     background-color: rgb(125, 158, 192);
```

Comment:
Please change the name of this file from manageFeedback.css to managefeedback.css as used for all other resources.